src/services/memory.service.ts

```
@@ -2428,6 +2467,519 @@ export class MemoryService {
+      await kuzuClient.executeQuery(tagDeleteQuery, {});
+
+      return { entities, count: entities.length };
+    }
+  }
+
+  private async processRepositoryScopedEntities(
+    kuzuClient: any,
+    entityTypes: string[],
+    repositoryName: string,
+    branch: string,
+    dryRun: boolean
```

ESLint

[error] Insert `,`

(prettier/prettier)